spec/config/tokens.spec.ts

```diff
@@ -0,0 +1,40 @@
+import TOKENS from 'config/tokens';
+import { isValidETHAddress } from 'libs/validators';
+
+describe('Tokens JSON', () => {
+  Object.keys(TOKENS).forEach(network => {
+    it(`${network} tokens array properly formatted`, () => {
+      const tokens = TOKENS[network];
+      const addressCollisionMap = {};
+      const symbolCollisionMap = {};
+
+      tokens.forEach(token => {
+        if (!isValidETHAddress(token.address)) {
+          throw Error(`Token ${token.symbol} has invalid contract address '${token.address}'`);
```

Comment:
Any reason for throwing errors instead of using `asserts`? Seems purely stylistic since CI should fail either way, but I'm curious.

Reply:
Maybe this is silly, but I wanted to reduce the overall test count (Hoping for some kind of performance gain, not to mention making it clear we don't really have 5000+ tests) which is why I didn't wrap these in `it`s. The throwing is so that you get a better error message than simply having a test called "ETH tokens list is properly formatted" failing, with no understanding of why.

I could convert these to individual `it`s, and use `expect` if that's preferred.

Reply:
I don't think it's a major issue either way, but it does go against convention and raises eyebrows.

Happy to merge either way.

Reply:
In that case I would prefer to leave as-is. The test failures you get this have very specific error messages that point to the problem, and should make it easier to find, i.e.

> Token GNT has invalid contract address '0xa74476443119A942dE498590Fe1f2454d7D4aC0doopsimsomeextracharacters'